PWGHF/HFL/DataModel/ElectronSelectionTable.h

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file ElectronSelectionTable.h
+/// \author Rashi Gupta <rashi.gupta@cern.ch>, IIT Indore
+/// \author Ravindra Singh <ravindra.singh@cern.ch>, IIT Indore
+
+#ifndef PWGHF_HFL_DATAMODEL_ELECTRONSELECTIONTABLE_H_
+#define PWGHF_HFL_DATAMODEL_ELECTRONSELECTIONTABLE_H_
+
+#include "Framework/AnalysisDataModel.h"
+
+namespace o2::aod
+{
+// definition of columns and tables forElectron Selection
+namespace hf_sel_electron
+{
+DECLARE_SOA_INDEX_COLUMN(Collision, collision);              //! CollisionID of Electrons
+DECLARE_SOA_INDEX_COLUMN(Track, track);                      //! Track ID of Electrons
+DECLARE_SOA_COLUMN(TrackEta, trackEta, float);               //!  Track Eta of Electrons
+DECLARE_SOA_COLUMN(TrackPhi, trackPhi, float);               //! Track Phi of Electrons
+DECLARE_SOA_COLUMN(TrackPt, trackPt, float);                 //! Track Transverse momentum of Electron
```

Comment:
```suggestion
DECLARE_SOA_COLUMN(EtaTrack, etaTrack, float);               //! pseudorapidity of the electron track
DECLARE_SOA_COLUMN(PhiTrack, phiTrack, float);               //! azimuth of the electron track
DECLARE_SOA_COLUMN(PtTrack, ptTrack, float);                 //! transverse momentum of the electron track
```
Please fix the naming everywhere. See the naming conventions.